### PR TITLE
Cleanup xmsannotator wrappers

### LINKED ISSUE #209: remove october_recetox_xmsannotator
and merge the needed functionality (if any) to the main tool

### PR DESCRIPTION
**Pull request by xtrojak**
Clean up of `xMSannotator` wrappers.

- removed the `october` prototype and kept only the main version
- removed the simple annotation wrapper
- made compatible with `recetox-aplcms` outputs
- added support for CSV format
- created minimal test case

Close #209

### CODE REVIEW

**Comment by xtrojak** [comment]
@hechth this is ready to go. The question is whether we want to wait for https://github.com/RECETOX/recetox-xMSannotator/issues/109 or do it later.

File: tools/recetox_xmsannotator/macros.xml [hunk 1]
```diff
@@ -1,74 +1,135 @@
...
-    </xml>
+<token name="@TOOL_VERSION@">0.10.0</token>
+
+<xml name="requirements">
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">r-recetox-xmsannotator</requirement>
+    </requirements>
+</xml>
+
+<xml name="creator">
```
Comment (hechth) [inline tools/recetox_xmsannotator/macros.xml:10]: @xtrojak please add yourself as well

File: tools/recetox_xmsannotator/recetox_xmsannotator_advanced.xml [hunk 2]
```diff
@@ -1,43 +1,43 @@
-<tool id="recetox_xmsannotator_advanced" name="RECETOX xMSannotator advanced" version="@TOOL_VERSION@+galaxy0">
+<tool id="recetox_xmsannotator_advanced" name="RECETOX xMSannotator" version="@TOOL_VERSION@+galaxy0">
```
Comment (hechth) [inline tools/recetox_xmsannotator/recetox_xmsannotator_advanced.xml:1]:
```suggestion
<tool id="recetox_xmsannotator_advanced" name="recetox-xMSannotator" version="@TOOL_VERSION@+galaxy0">

```

File: tools/recetox_xmsannotator/recetox_xmsannotator_advanced.xml [hunk 3]
```diff
@@ -89,21 +85,6 @@
...
-                </help>
-            </param>
-            <param name="boost_compounds" type="data" format="csv" optional="true">
-                <label>Validated compounds score boosting (optional)</label>
-                <help>
-                    Table of previously validated compounds to boost their scores and confidence levels.
-                    The 1st column of the table must contain IDs of compounds.
-                    The optional 2nd and 3rd columns may contain mz values and retention times.
-                </help>
-            </param>
```
Comment (hechth) [inline tools/recetox_xmsannotator/recetox_xmsannotator_advanced.xml:106]: What happened with these parameters? Do we just not support boosting anymore at this point?

Comment (xtrojak) [reply tools/recetox_xmsannotator/recetox_xmsannotator_advanced.xml:106]: Yeah, these were removed in the october version of xMSannotator.

**Comment by hechth** [comment]
@xtrojak so we agreed we still add the bio.tools crossref etc. here and then merge this PR?

**Review by hechth — APPROVED** [review]

**Review by hechth — APPROVED** [review]

**Pull request merged**